package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "raptor",
```

might be better to name screwdriver-raptor to fit in naming conventions for our other packages (see https://github.com/screwdriver-cd/models/blob/master/package.json#L2)

```suggestion
  "name": "screwdriver-raptor",
```